docs/usage/quickstart.rst

```diff
@@ -159,49 +162,55 @@ first run)::
     Launching MetalK8s shell environment. Run 'exit' to quit.
     (metal-k8s) $
 
-Now we're all set to deploy a cluster::
+Deploy the Cluster
+------------------
 
-    (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster -b playbooks/deploy.yml
+Run the following command to deploy the cluster::
 
-Grab a coffee and wait for deployment to end.
+  (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster/hosts
```

Comment:
Dropped `-b` and `playbooks/deploy.yml` which are required.

Reply:
I'll blame that on a miscommunication on this side of the Atlantic. Can you put them back in, or is there something I need to do here?